### PR TITLE
change(PR) readme

### PR DESCRIPTION
**Pull request by ImreDW**
gave the repository purpose

### CODE REVIEW

**Comment by justagitcookie** [comment]
the cake is a lie.

**Comment by ImreDW** [comment]
*the cake is delicious!

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # hello-world
 Temp Upload to get more familiar with github.
 
-This repository serves no purpose besides me learning how to branch and do more advanced git stuff than i usually use.
```
Comment (justagitcookie) [inline README.md]: this is a fantastic change, i appreciate it!

Comment (justagitcookie) [reply README.md]: ADD AMAZING :)

Comment (ImreDW) [reply README.md]: ok!

**Pull request merged**